### PR TITLE
feat(changes): provide POC for (near) real-time changes notification

### PR DESCRIPTION
**Pull request by yhuard**
This is a WIP, please DO NOT merge.

EDIT 12/10/2016: you can now merge it.

### CODE REVIEW

File: services/changes/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,40 @@
+0 info it worked if it ends with ok
```
Comment (kalinchernev) [inline services/changes/npm-debug.log]: this file is probably added unintentionally

Comment (yhuard) [reply services/changes/npm-debug.log]: Good catch, thanks

**Comment by yhuard** [comment]
How to test: 
- simplest solution: run `docker-compose build` and then `docker-compose up`. The tests should be passing (63/63). Go to your Fauxton interface, edit documents, save them. Check that cope-changes effectively detect those changes.

**Review by kalinchernev — APPROVED** [review]
Just adding a green stamp :)

**Pull request merged**